### ISSUE TITLE
Erreur lors de la configuration du SMTP

### ISSUE THREAD

**Issue by dNicolle** [edissyum/opencapture#1295]
Bonjour,

Lorsque je tente de valider ma configuration SMTP, j'ai une erreur qui me dit
"Erreur de requête</b> : Les données obligatoires sont manquantes ou incorrectes : 'value' n'est pas de type 'string'"

Et cela que je remplisse les 3 premiers champs ou l'ensemble des champs.

Les logs du fichier OpenCapture.log ne se remplissent pas sur cette action.
Je ne suis pas administratrice du serveur où il est installé, je n'ai pas accès aux logs Apache.
Je les aient demandées.

**Comment by dNicolle** [comment]
3.0.6, pardon, je l'indique aussi dans l'autre issue

**Comment by nathan30** [comment]
Le souci est normalement corrigé depuis la version 3.1.2

Avez-vous la possibilité de procéder à une mise à jour du produit ?

**Comment by dNicolle** [comment]
Bonjour,
Suite à la mise à jour, je n'ai plus cette erreur.
Cependant, j'ai un soucis de version SSL alors que je suis en TLS. (SMTP OVH pro)
`Erreur lors de la connexion SMTP</b> : [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:992)`

**Comment by dNicolle** [comment]
Oui, le port 25 non sécurité (authentification refusée) et le port 465 SSL ne sont pas non plus fonctionnels (timeout)

**Comment by nathan30** [comment]
Bonjour. En effet, un souci avec le TLS était présent. Un fix est sorti : https://github.com/edissyum/opencapture/commit/9b3a508cb23a55b100e0102084d430e2b5f91f1e
Vous avez le choix de la déployer manuellement sur votre instance, sinon d'attendre la sortie du prochain tag

Cordialement
